doc/classes/Tween.xml

```diff
@@ -350,6 +350,11 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="tween_all_completed">
+			<description>
+				Emitted when all processes in a tween ends.
```

Comment:
Should be "end" here for plural, not "ends".

Reply:
Fixed